astropy/modeling/core.py

```diff
@@ -2315,6 +2407,113 @@ def _custom_model_wrapper(func, fit_deriv=None):
     return type(model_name, (FittableModel,), members)
 
 
+def render_model(model, arr=None, coords=None, copy=True):
+    """
+    Evaluates a model on an input array. Evaluation is limited to
+    a bounding box if the `Model.bounding_box` attribute is set.
+
```

Comment:
You could add one or two examples in an `Examples` section here and / or link to the section with the examples in the high-level docs.

See http://sphinx-doc.org/markup/inline.html#cross-referencing-arbitrary-locations for a desription of Sphinx labels and references.